src/components/ContactCard/ContactIdentity.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import PropTypes from "prop-types";
+import ContactBadge from "../ContactBadge";
+import contactPropTypes from "../ContactPropTypes";
+
+const ContactIdentity = ({ name, groups }) => (
+  <div className="contact-identity">
+    <ContactBadge firstname={name.givenName} lastname={name.familyName} />
+    <div>
+      <h1 className="contact-identity__title">
+        {`${name.namePrefix || ""} ${name.givenName ||
+          ""} ${name.additionalName || ""} ${name.familyName ||
+          ""} ${name.nameSuffix || ""}`.trim()}
+      </h1>
+      <ContactGroups groups={groups} />
+    </div>
```

Comment:
Maybe you could wrap the avatar and the title in a Media so that they are vertically aligned ?

Reply:
Yeah this is part of the header that I need to rework, using the `Avatar` component from cozy-ui. Once I do that they will be properly aligned 👍

Reply:
J'ai vu le même code dans une autre PR https://github.com/cozy/cozy-contacts/pull/30/files#diff-08ee3ae2be37d1f20ca29d9a68d8a162R27